packages/@atjson/document/src/index.ts

```diff
@@ -65,19 +65,48 @@ export {
   UnknownAnnotation
 };
 
+export function getConverterFor(from: typeof Document, to: typeof Document): never | ((doc: Document) => Document) {
+  let exports = (typeof window !== 'undefined' ? window : global) as any;
+
+  let converters = exports.__atjson_converters__;
+  let converter = converters ?
+    converters[from.contentType] ?
+      converters[from.contentType][to.contentType] :
+    null :
+  null;
+  if (converter == null) {
+    throw new Error(`🚨 There is no converter registered between ${from.name} and ${to.name}.\n\nRegister a converter for this:\n\n${from.name}.defineConverterTo(${to.name}, doc => {\n  // ❤️ Write your converter here!\n  return doc;\n});`);
```

Comment:
It might be worth including a "if you expected this to work, maybe you haven't `import`'ed the converter?" bit, in case people end up thinking they need to write converters that just haven't been imported.